generator/templates/drivers/inc/testfamily.hpp.j2

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "static.hpp"
+#include "bus.hpp"
+
+#include <stdint.h>
+#include <ctime>
+#include <chrono>
+#include <vector>
+#include <cstring>
+#include "clock.hpp"
+
+
+namespace CANlib {
+
+
+class CAN {
```

Comment:
Feel like this should be named something like `TestCAN` or `TestDriver` or something